streams/src/test/java/org/apache/kafka/streams/kstream/internals/suppress/KTableSuppressProcessorTest.java

```diff
@@ -461,10 +468,6 @@ public boolean matches(final Object item) {
         };
     }
 
-    private static <K> SuppressedInternal<K> getImpl(final Suppressed<K> suppressed) {
-        return (SuppressedInternal<K>) suppressed;
-    }
-
```

Comment:
inline unnecessary method. I must have made this when I thought that the cast would cause a compiler warning, but it does not.